plutus-use-cases/src/Language/PlutusTx/Coordination/Contracts/CrowdFunding.hs

```diff
@@ -142,20 +144,35 @@ scriptInstance cmp = Scripts.Validator @CrowdFunding
 {-# INLINABLE validRefund #-}
 validRefund :: Campaign -> PubKey -> PendingTx -> Bool
 validRefund campaign contributor ptx =
+    -- Check that the transaction falls in the refund range of the campaign
     Interval.contains (refundRange campaign) (pendingTxValidRange ptx)
+    -- Check that the transaction is signed by the contributor
     && (ptx `V.txSignedBy` contributor)
 
 {-# INLINABLE validCollection #-}
 validCollection :: Campaign -> PendingTx -> Bool
 validCollection campaign p =
+    -- Check that the transaction falls in the collection range of the campaign
     (collectionRange campaign `Interval.contains` pendingTxValidRange p)
-    && (valueSpent p `VTH.geq` campaignTarget campaign)
+    -- Check that the transaction is trying to spend more money than the campaign
+    -- target (and hence the target was reached)
+    && (valueSpent p `Value.geq` campaignTarget campaign)
+    -- Check that the transaction is signed by the campaign owner
     && (p `V.txSignedBy` campaignOwner campaign)
 
 {-# INLINABLE mkValidator #-}
+-- | The validator script is of type 'CrowdfundingValidator', and is
+-- additionally parameterized by a 'Campaign' definition. This argument is
+-- provided by the Plutus client, using 'Ledger.applyScript'.
+-- As a result, the 'Campaign' definition is part of the script address,
+-- and different campaigns have different addresses. The Campaign{..} syntax
```

Comment:
```suggestion
-- and different campaigns have different addresses.
```

Reply:
`RecordWildCards` isn't used anymore (yeah I was the one who put it in in the first place, but I saw the errors of my ways ;) )